py/server/deephaven/updateby.py

```diff
@@ -1228,11 +1230,12 @@ def rolling_std_tick(cols: Union[str, List[str]], rev_ticks: int, fwd_ticks: int
 
 def rolling_std_time(ts_col: str, cols: Union[str, List[str]], rev_time: Union[int, str],
                        fwd_time: Union[int, str] = 0) -> UpdateByOperation:
-    """Creates a rolling standard deviation UpdateByOperation for the supplied column names, using time as the windowing unit. This
-    function accepts nanoseconds or time strings as the reverse and forward window parameters. Negative values are
-    allowed and can be used to generate completely forward or completely reverse windows. A row containing a null in
-    the timestamp column belongs to no window and will not be considered in the windows of other rows; its output will
-    be null.
+    """Creates a rolling sample standard deviation UpdateByOperation for the supplied column names, using time as the
+    windowing unit. Sample standard deviation is computed using `Bessel's correction <https://en.wikipedia.org/wiki/Bessel%27s_correction>`_,
+    which ensures that the sample variance will be an unbiased estimator of population variance.
+    This function accepts nanoseconds or time strings as the reverse and forward window parameters. Negative values are
+    allowed and can be used to generate completely forward or completely reverse windows. A row containing a null in the
+    timestamp column belongs to no window and will not be considered in the windows of other rows; its output will be null.
```

Comment:
In all of these places, the bessel part should be a separate paragraph after the original paragraph.